gateway/1.4/modules/ROOT/pages/_partials/task-shared-storage-config.adoc

```diff
@@ -8,19 +8,19 @@ Production workflows should use https://redis.io/[Redis^], though defining it is
 You configure shared storage via a custom YAML file.
 
 The following steps demonstrate Redis-based storage support for Flex Gateway running in a Docker container, as a Linux Service,
-or as a Kubernetes Ingress controller.
+or as a Kubernetes deployment.
```

Comment:
On a second though, I think you are right. I was misinterpreting the term, sorry. `Kubernetes cluster` is fine in this context

Reply:
reverted